infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/main.tf

```
@@ -146,5 +148,6 @@ module "prow_build_nodepool" {
   disk_size_gb    = 200
   disk_type       = "pd-standard"
   service_account = module.prow_build_cluster.cluster_node_sa.email
+  enable_node_local_dns_cache = var.enable_node_local_dns_cache
```

I may be wrong but I think it's : 
```suggestion
  enable_node_local_dns_cache = lcoal.enable_node_local_dns_cache
```